src/app/app.module.ts

```diff
@@ -2,15 +2,47 @@ import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
 
 import { AppComponent } from './app.component';
+import { HeaderComponent } from './header/header.component';
```

Comment:
Please use the material components that you will be using, another removed. Read the article: https://indepth.dev/posts/1191/stop-using-shared-material-module